### PR TITLE
Refactor of snowpark registry

### PR DESCRIPTION
**Pull request by sfc-gh-jsikorski**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added new automated tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the section below.

### Changes description
Refactor for snowpark registry commands

### CODE REVIEW

File: tests_integration/test_registry.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from tests_integration.conftest import runner
+
+
+def test_token(runner):
```
Comment (sfc-gh-astus) [inline tests_integration/test_registry.py:6]:
```
@pytest.mark.skip(reason="Not yet enabled on our account")
@pytest.mark.integration
```

are missing

File: tests_integration/test_registry.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    result = runner.invoke_integration(["snowpark", "registry", "token"])
+
+    assert result.exit_code == 0
```
Comment (sfc-gh-astus) [inline tests_integration/test_registry.py:9]: and result.json should contains not empty fields `token` and `expires_in`

File: tests_integration/test_registry.py (hunk 1 above)
Comment (sfc-gh-jsikorski) [reply tests_integration/test_registry.py:6]: Done

File: tests/test_snow_connector.py [hunk 3]
```diff
@@ -129,17 +129,20 @@ def test_upload_file_to_stage(_, snapshot, create_stage, stage_name):
...
         }
     }
-    result = runner.invoke(["snowpark", "registry", "token"])
+    result = runner.invoke(["snowpark", "registry", "token", "--format", "JSON"])
     assert result.exit_code == 0, result.output
-    assert result.stdout == '{"token": "token1234", "expires_in": 42}'
+    assert (
+        result.stdout
+        == '[\n  {\n    "token": "token1234",\n    "expires_in": 42\n  }\n]\n'
+    )
```
Comment (sfc-gh-pjob) [inline tests/test_snow_connector.py:145]:
```suggestion
    assert json.loads(result.stdout) == [{"token": "token1234", "expires_in": 42}]
```

Comment (sfc-gh-pjob) [reply tests/test_snow_connector.py:145]: + required import of `json`

File: src/snowcli/cli/snowpark/registry/commands.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+app = typer.Typer(
+    context_settings=DEFAULT_CONTEXT_SETTINGS, name="registry", help="Manage registry"
+)
+
+
+@app.command("token")
+@with_output
+@global_options_with_connection
+def token(environment: str = ConnectionOption, **options):
```
Comment (sfc-gh-astus) [inline src/snowcli/cli/snowpark/registry/commands.py:18]: `environment` should not be needed with global_options

**Review by sfc-gh-turbaszek — DISMISSED** [review]

**Review by sfc-gh-astus — APPROVED** [review]

File: tests/test_snow_connector.py (hunk 3 above)
Comment (sfc-gh-jsikorski) [reply tests/test_snow_connector.py]: Done

**Review by sfc-gh-astus — APPROVED** [review]

**Pull request merged**